src/index.ts

```diff
@@ -306,8 +313,19 @@ class Geckoboard {
     return new Dataset(schema);
   }
 
-  ping(): Promise<void> {
-    return Promise.resolve();
+  async ping(): Promise<void> {
+    const auth = btoa(`${this.apiKey}:`);
+    const res = await fetch('https://api.geckoboard.com', {
+      headers: {
+        Authorization: `Basic ${auth}`,
```

Comment:
Let's make sure we don't forget to send a user header containing the library version (can be a future PR) https://github.com/geckoboard/geckoboard-node/blob/44a0d138875cc3dd83021ebb34d15a3fa248927f/lib/index.js#L7

Reply:
Ahh yes, I mean to include that, thanks for reminder!